test/cli/compiler/test-externals.js

```diff
@@ -1,29 +1,31 @@
 const test = require("tape");
 const testUtils = require("../../../bin/tools/utils");
 const fsPromises = testUtils.fsPromises;
+require("process").chdir(__dirname);
 
-test("test externals", async assert => {
+test("test externals", async t => {
   try {

```

Comment:
It's the tape convention, which is used throughout the tape documentation.  The object passed into the test function contains assert fuctions, but many other ones to, which are nonsensical if called `assert` - for example, `assert.end()` doesn't assert that the test ends, it signals that it ends.

Reply:
hmm, I thought that I'd picked up the `assert` from tape when i wrote this; but I think that the issue remains, which is that `t` is undescriptive, confusing because it can be used (and has been used) for other things beginning with "t" (like `this`).

Reply:
See https://github.com/substack/tape/#example - I assume that most users of tape follow this convention and thus I don't think it gets confused, as opposed if you call it `assert`. Not convinced?

Reply:
Among other methods in the passed object t.plan(n), t.end(err), t.fail(msg), t.pass(msg), t.comment() and t.timeoutAfter(ms) all do not assert anything. We could also use a shadowed `test` or `tst` if you prefer, but I would move away from `assert`.

Reply:
i'd already checked that webpage before i commented.  

I'm not convinced because I'm really concerned about other tape users, especially as we're planning on ditching tape in exchange for everything being part of qooxdoo test framework, and IMHO just because `tape` authors would like to suggest that `t` should be reserved for `tape`, that's not something which is established - quite the opposite, there are 62 uses of `var t=this` in framework and google shows that it's a common idiom.

also, imho mass changes to code without a clear reason isn't a good idea - this applies to reformatting as well as these kind of changes.

Reply:
I have a different opinion, but it is your code, and you should like it if you work on it!  I reverted.